### ISSUE TITLE
[Week 2] 발제자 피드백 (나송주)

### ISSUE THREAD

**Issue by SongjuNa** [team-epoch/EPOCH_4th_TASK#29]
### [발제 후 들어온 질문]

**1. AlexNet에서 효과적이었던 LRN의 효과가 VGGNet에서는 미미했던 이유?**

활성화 함수로 ReLU를 모든 층에서 사용했고, 작은 3x3 필터를 사용했기 때문입니다. VGGNet에서는 dropout과 weight decay 방식만으로도 충분히 안정적이었기 때문에 최종 성능 검증 때 LRN을 제외했습니다. 실제 실험 결과에서 모델 A와 A-LRN을 비교했을 때, 오히려 성능 향상보다는 악화로 이어지는 것을 확인할 수 있습니다. 이후, BatchNormalization이 등장하면서 LRN은 사실상 쓰이지 않았다고 합니다! 우리가 모두 LRN을 처음 접해본 개념일만큼 낯설다는 게 이런 이유가 아닐까 싶습니다.

**2. training할 때 랜덤 초기화 기법을 이용해 pre training 없이도 초기화를 시켰는데 그게 무슨 의미가 있는지?**

VGGNet은 처음에는 얕은 네트워크(A)로 pre-training 후, 깊은 네트워크를 초기화했지만, 이후 Xaiver 같은 랜덤초기화 기법을 적용하면 pre-training 없이 안정적인 학습이 가능하다는 것을 확인했다고 합니다. Xaiver 초기화는 forward와 backward 출력 시, 출력과 gradient 분산이 유지되도록 weight를 일정 범위에서 균등하게 샘플링하는 기법이라고 합니다. (기존 alexnet, vgg 초창기에 사용하던 초기화 기법은 gradient 불안정 문제가 있었고, Xaiver가 이를 해결함)
--------------------------------------------------------------------------------------------------
### [개인적으로 다같이 논의해보고 싶었던 점]

** 작은 필터 여러 번 사용하는 것이 유리하다고 했는데, 왜 하필 작은 필터로 1x1도, 2x2도 아닌 3x3 사이즈를 사용했을까?**

1x1필터는 채널 간 선형 결합만 가능하기 때문에 context는 전혀 학습할 수 없다. 따라서 Inception 같은 보조 역할만 가능하고, 메인 필터로 사용하기에는 부족하다.
2x2필터는 receptive field가 너무 작아 상하좌우 방향성을 구분하기 힘들다. 따라서 sliding할 때, 중심점이 없어 공간적 구조를 표현하기에 부적합하다.
3x3필터는 상하좌우와 중심점을 포착하기에 가장 적합한 receptive field이고, 그래서 공간을 담을 수 있는 최소 단위의 필터 사이즈이다. 이 외의 장점은 논문 내용에 충분히 담겨있고, 최소의 적합한 필터의 크기가 3x3이기 때문에 채택한 것 같다.
